docs/dev-guide/contributing/dev_setup.rst

```diff
@@ -10,7 +10,7 @@ For the Impatient
 These instructions will create a developer install of Pulp on a dedicated
 development instance.
 
-* Start a Fedora 20 x86_64 instance that will be dedicated for development with
+* Start a RHEL 7 or Fedora 20 x86_64 instance that will be dedicated for development with
```

Comment:
Works with the beta repos, but then there is a problem running tests

http://sprunge.us/PHXY